nodeset/Opc.Ua.PlasticsRubber.GeneralTypes.NodeSet2.xml

```diff
@@ -0,0 +1,14109 @@
+        </References>
+        <Value>
+            <uax:String></uax:String>
+        </Value>
+    </UAVariable>
+    <UAVariable DataType="String" ParentNodeId="ns=1;i=5023" NodeId="ns=1;i=6162" BrowseName="1:Name">
+        <DisplayName>Name</DisplayName>
+        <Description>Name of the user</Description>
+        <References>
+            <Reference ReferenceType="HasTypeDefinition">i=68</Reference>
+            <Reference ReferenceType="HasModellingRule">i=78</Reference>
+            <Reference ReferenceType="HasProperty" IsForward="false">ns=1;i=5023</Reference>
+        </References>
+        <Value>
+            <uax:String></uax:String>
+        </Value>
+    </UAVariable>
+    <UAVariable DataType="String" ParentNodeId="ns=1;i=5023" NodeId="ns=1;i=6163" BrowseName="1:UserLevel">
+        <DisplayName>UserLevel</DisplayName>
+        <Description>Level of the user</Description>
+        <References>
+            <Reference ReferenceType="HasTypeDefinition">i=68</Reference>
+            <Reference ReferenceType="HasModellingRule">i=80</Reference>
+            <Reference ReferenceType="HasProperty" IsForward="false">ns=1;i=5023</Reference>
+        </References>
+        <Value>
+            <uax:String></uax:String>
+        </Value>
+    </UAVariable>
+    <UAVariable DataType="String" ParentNodeId="ns=1;i=5023" NodeId="ns=1;i=6164" BrowseName="1:UserRole">
+        <DisplayName>UserRole</DisplayName>
+        <Description>Role of the user</Description>
+        <References>
+            <Reference ReferenceType="HasTypeDefinition">i=68</Reference>
+            <Reference ReferenceType="HasModellingRule">i=80</Reference>
+            <Reference ReferenceType="HasProperty" IsForward="false">ns=1;i=5023</Reference>
+        </References>
+        <Value>
+            <uax:String></uax:String>
+        </Value>
+    </UAVariable>
+    <UAObjectType IsAbstract="true" NodeId="ns=1;i=1014" BrowseName="1:MachineModeChangeLogType">
+        <DisplayName>MachineModeChangeLogType</DisplayName>
+        <Description>The MachineModeChangeLogType is used for logging changes of the machine mode</Description>
+        <References>
+            <Reference ReferenceType="HasSubtype" IsForward="false">ns=1;i=1011</Reference>
+            <Reference ReferenceType="HasProperty">ns=1;i=6175</Reference>
+            <Reference ReferenceType="HasProperty">ns=1;i=6176</Reference>
+        </References>
+    </UAObjectType>
+    <UAVariable DataType="MachineModeEnumeration" ParentNodeId="ns=1;i=1014" NodeId="ns=1;i=6175" BrowseName="1:NewMachineMode">
+        <DisplayName>NewMachineMode</DisplayName>
+        <Description>New machine mode</Description>
+        <References>
+            <Reference ReferenceType="HasTypeDefinition">i=68</Reference>
+            <Reference ReferenceType="HasModellingRule">i=78</Reference>
+            <Reference ReferenceType="HasProperty" IsForward="false">ns=1;i=1014</Reference>
+        </References>
+    
```

Comment:
Adding the `print(e)` suggested change indicates that it still struggles to import this node:

```
WARNING:asyncua.common.xmlimporter:failure adding node NodeData(nodeid:NodeId(Identifier=6318, NamespaceIndex=8, NodeIdType=<NodeIdType.Numeric: 2>))
WARNING:root:Import Error - 'str' object has no attribute 'NamespaceIndex'
WARNING:asyncua.common.xmlimporter:Model is missing: http://opcfoundation.org/UA/PlasticsRubber/GeneralTypes/ - Version: 1.03.01 - PublicationDate: 2022-07-13T00:00:00Z or newer
WARNING:root:Import Error - Server doesn't satisfy required XML-Models. Import them first!
```

Reply:
https://github.com/DrHaraldWeber/Sample-Server-asyncio/blob/395c281faec753bd651e546f244b2641f518e619/nodeset/Opc.Ua.PlasticsRubber.GeneralTypes.NodeSet2.xml#L4954
Don't know why GH lists other code above, although marked the same spot.